### ISSUE TITLE
BadRequestError: failed to natively upload attachment

### ISSUE THREAD

**Issue by ganthern** [tutao/tutanota#1007]
File sizes don't match: 610721 uploaded vs 610247 announced.

```
Client: android app
Type: FREE
Tutanota version: 3.44.2
Timestamp (UTC): Mon, 07 Jan 2019 13:33:54 GMT
User agent:
Mozilla/5.0 (Linux; Android 9; Poco F1 Build/PQ1A.181105.017.A1; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/66.0.3359.158 Mobile Safari/537.36


Error message:
400: LVcWFNd----1 | PUT https://mail.tutanota.com/rest/tutanota/filedataservice?fileDataId=LVcWEkP----1 failed to natively upload attachment


Stacktrace:
BadRequestError: 400: LVcWFNd----1 | PUT https://mail.tutanota.com/rest/tutanota/filedataservice?fileDataId=LVcWEkP----1 failed to natively upload attachment
at handleRestError (file:///android_asset/tutanota/dist/main-boot.js:1:245780)
at file:///android_asset/tutanota/dist/worker.js:1:230515
at a (file:///android_asset/tutanota/dist/libs.js:26:73165)
at F._settlePromiseFromHandler (file:///android_asset/tutanota/dist/libs.js:26:46542)
at F._settlePromise (file:///android_asset/tutanota/dist/libs.js:26:47342)
at F._settlePromise0 (file:///android_asset/tutanota/dist/libs.js:26:48041)
at F._settlePromises (file:///android_asset/tutanota/dist/libs.js:26:49393)
at F._fulfill (file:///android_asset/tutanota/dist/libs.js:26:48411)
at Object.worker19 (file:///android_asset/tutanota/dist/libs.js:26:37178)
at Queue.value (file:///android_asset/tutanota/dist/main-boot.js:1:278936)
at Queue._transport._transport.onmessage (file:///android_asset/tutanota/dist/main-boot.js:1:278552)
```

**Comment by ganthern** [comment]
https://www.reddit.com/r/tutanota/comments/ad70na/attaching_photo_crashes_the_app/

**Comment by ganthern** [comment]
optimistically declaring this as solved by d96bcd8bea5c119d3873ceee8cfa0422bbeed6a5
